app/models/api_token.rb

```diff
@@ -0,0 +1,10 @@
+class APIToken < ApplicationRecord
+  has_secure_token :token, length: 32
+  encrypts :token, deterministic: true
+
+  belongs_to :lead_provider
+
+  validates :lead_provider, presence: { message: "Lead provider must be specified" }
```

Comment:
can we add the error messages to en.yml maybe?

Reply:
There is no `en.yml` in RECT as it was decided to use inline error messages

Reply:
Think it should be ok to create the en.yml file

Reply:
We could re-add it, but I mean it was intentionally removed as a team decision before we got here so we'd need to rehash that discussion 🙂  https://github.com/DFE-Digital/register-early-career-teachers-public/commit/0ffc5448c0266e393d461d21c006848e934fe748

Reply:
Yeah it's easier for everyone when the messages are alongside the errors.

Reply:
A bonus benefit is that it makes spotting `"is missing"` errors much easier to spot.